db/migrate/20150319212424_migrate_ncr_to_models.rb

```diff
@@ -0,0 +1,69 @@
+class MigrateNcrToModels < ActiveRecord::Migration
+  class Property < ActiveRecord::Base
+  end
+  class Cart < ActiveRecord::Base
+    has_many :properties, -> { where hasproperties_type: 'Cart' },
+              foreign_key: "hasproperties_id"
+    belongs_to :proposal
+    def props
+      Hash[self.properties.collect{|p| [p.property, YAML::load(p.value)]}]
+    end
+  end
+  class Proposal < ActiveRecord::Base
+    has_one :cart
+  end
+  class NcrWorkOrder < ActiveRecord::Base
+    has_many :proposals, -> { where client_data_type: 'Ncr::WorkOrder' },
+              foreign_key: "client_data_id"
+  end
```

Comment:
Minor: I've been prefixing the migration-only models w/ `Temp` so that I don't accidentally reference the normal ones.